samples/bootloader/CMakeLists.txt

```diff
@@ -10,3 +10,41 @@ find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
 project(bootloader)
 
 zephyr_library_sources(src/main.c)
+
+if(CONFIG_SOC_SERIES_NRF54LX)
+set(OUTPUT_FILE_PATH ${APPLICATION_BINARY_DIR}/${DEFAULT_IMAGE}/zephyr)
```

Comment:
erm.... yeah that's a no :rofl: look how wifi does it and follow that, kconfig: https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/Kconfig.wifi cmake: https://github.com/nrfconnect/sdk-nrf/blob/main/sysbuild/CMakeLists.txt#L671 https://github.com/nrfconnect/sdk-nrf/blob/main/cmake/sysbuild/nrf700x.cmake